client/src/components/CalcIcon.js

```diff
@@ -0,0 +1,29 @@
+import styled from '@emotion/styled';
+
+const CalcIcon = styled.button`
+  height: 90px;
+  width: 90px;
+  border: solid ${(props) => calcColor[props.color].bordercolor};
+  border-width: 3px;
+  border-radius: 100%;
+  font-size: 30px;
+  color: ${(props) => calcColor[props.color].textcolor};
+  background: none;
+  font-family: 'Rubik', sans-serif;
+  margin-bottom: 25px;
+`;
+
+const calcColor = {
+  perfect: { bordercolor: '#ff0084', textcolor: '#ff0084' },
+  veryhigh: { bordercolor: '#ff00f2', textcolor: '#ff00f2' },
```

Comment:
camelCase in this file for all properties

```suggestion
  veryHigh: { borderColor: '#ff00f2', textColor: '#ff00f2' },
```